global.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
It is something like a solution file. It is help to resolve the package references on solution level.